package.json

```diff
@@ -30,6 +30,7 @@
     "autoprefixer": "10.4.14",
     "big.js": "^6.2.1",
     "eslint-config-next": "13.4.7",
+    "framer-motion": "^11.3.18",
```

Comment:
Why do you need framer-motion library? Can you list the usecases? 

The library is pretty big (2.19 MB). 

Do you know if the full library is loaded or only the components you use?

Reply:
I first used framer-motion for its easy to implement intersection observer (to start increment counter when counter is in view) 
but finally found a lighter library, "react-countup", which also implements intersection observer pretty easily